### ISSUE TITLE
server not running on Python 3.5

### ISSUE THREAD

**Issue by brianmay** [sshuttle/sshuttle#34]
Basically we get this error from the code in ssuttle/ssh.py:

```
Python 3.5.0 (default, Nov 16 2015, 18:39:12) 
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> exec compile(sys.stdin.read(%d), "assembler.py", "exec")
  File "<stdin>", line 1
    exec compile(sys.stdin.read(%d), "assembler.py", "exec")
               ^
SyntaxError: invalid syntax
```

**Comment by brianmay** [comment]
Actually that line has a copy and paste error. `%d` should be 380. Still gives me a syntax error though.

Oh, maybe exec is a function, and requires brackets now.

**Comment by brianmay** [comment]
Yes, that looks healthier. Python 3.4 won't work though. Ideally the auto-detect code would somehow fall back to Python 2.7 if the server only has Python 3.4.

**Comment by brianmay** [comment]
Fixed in 65e81d51c6968b46bd035dada54213b654a580b8.
